test/ops/reduction_ops_test.py

```diff
@@ -378,7 +379,7 @@ def testEmptyGradients(self):
   def testDegenerate(self):
     # TODO: Remove this skip once the resource zeroing bug has been fixed
     # TFDML #41044841
-    if should_skip_test(".*UHD Graphics 630.*"):
+    if len(argv) >= 2 and should_skip_test(".*UHDGraphics630.*", argv[1]):
```

Comment:
Can't we pass spaces via the command line by surrounding the adapter name with double quotes?